components/Cards/examples/EditReorderCollectionViewController.swift

```diff
@@ -121,6 +121,10 @@ class EditReorderCollectionViewController: UIViewController,
 
     MDCCardThemer.applyScheme(cardScheme, toCardCell: cell)
     cell.isSelectable = (toggle == .edit)
+
+    cell.isAccessibilityElement = true
+    cell.accessibilityLabel = "Cell"
```

Comment:
I think it would be more helpful if the cards themselves had unique content.  Maybe we can name them after their colors (do the colors vary)? Perhaps they could have a text label? What do you think?

Reply:
Working on it. Will be available in a separate PR.